app/models/tag.rb

```diff
@@ -122,15 +122,8 @@ def self.for_names(tag_names, ns)
     where(:name => fq_tag_names)
   end
 
-  def self.find_by_classification_name(name, region_id = Classification.my_region_number,
-                                       ns = Classification::DEFAULT_NAMESPACE, parent_id = 0)
-    in_region(region_id).find_by(:name => Classification.name2tag(name, parent_id, ns))
-  end
-
-  def self.find_or_create_by_classification_name(name, region_id = Classification.my_region_number,
-                                                 ns = Classification::DEFAULT_NAMESPACE, parent_id = 0)
-    tag_name = Classification.name2tag(name, parent_id, ns)
-    in_region(region_id).find_by(:name => tag_name) || create(:name => tag_name)
+  def self.find_by_classification_name(name)
```

Comment:
Is this method needed anymore?  I searched the code across all of the repos and the only usages seem to be the ones that you've already deleted elsewhere in this PR (and of course the specs, which can also be deleted if the method is deleted).

Reply:
there is only 1 use of this method in [miq consumption](https://github.com/ManageIQ/manageiq-consumption/blob/master/app/models/manageiq/showback/data_rollup.rb#L171)